src/Classes/Config.php

```diff
@@ -80,6 +80,13 @@
     public static $schedule_url = '//ury.org.uk/schedule';
 
     /**
+     * The base URL of the radio home pages.
+     *
+     * @var string
+     */
+    public static $home_url = '//ury.org.uk/';
```

Comment:
@AJDurant How do I go about setting that in the local config? Ie how is that generated?